demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -90,6 +90,8 @@ def is_condition_branches_handled(self):  # type: () -> bool
         Return:
             bool. if the Playbook handles all condition branches correctly.
         """
+        if 'TestPlaybooks' in self.file_path:
```

Comment:
verify we have test that cover this condition

Reply:
@hod-alpert did you verify that?

Reply:
Yes, I added a test for that ` test_unhandled_condition_skipping`